test/OpenTelemetry.Tests/Metrics/MemoryEfficiencyTests.cs

```diff
@@ -29,7 +29,7 @@ public class MemoryEfficiencyTests
         [InlineData(AggregationTemporality.Delta)]
         public void ExportOnlyWhenPointChanged(AggregationTemporality temporality)
         {
-            using var meter = new Meter(Utils.GetCurrentMethodName() + temporality, "1.0");
+            using var meter = new Meter(Utils.GetCurrentMethodName() + temporality);
```

Comment:
Maybe adding a `.`? (or string interpolation)

```suggestion
            using var meter = new Meter(Utils.GetCurrentMethodName() + "." + temporality);
```

Reply:
Sure!